src/libmetis/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+target_sources(METIS_METIS PRIVATE
+		auxapi.c
+		balance.c
+		bucketsort.c
+		checkgraph.c
+		coarsen.c
+		compress.c
+		contig.c
+		debug.c
+		fm.c
+		fortran.c
+		frename.c
+		gklib.c
+		graph.c
+		initpart.c
+		kmetis.c
+		kwayfm.c
+		kwayrefine.c
+		mcutil.c
+		mesh.c
+		meshpart.c
+		minconn.c
+		mincover.c
+		mmd.c
+		ometis.c
+		options.c
+		parmetis.c
+		pmetis.c
+		refine.c
+		separator.c
+		sfm.c
+		srefine.c
+		stat.c
+		timing.c
+		util.c
+		wspace.c
+)
+target_link_libraries(METIS_METIS PRIVATE GKlib::GKlib)
+
+if (TARGET GKlib_GKlib)
+	target_compile_options(GKlib_GKlib PRIVATE -fPIC)
```

Comment:
use POSITION_INDEPENDENT_CODE

Reply:
Indeed, it is marked as a temp commit because I am not sure how to properly handle it. Should `GKlib` become a shared library and be installed together with `METIS`, or remain a static, but patch it to be PIC. This is the first time I've encountered where this has become an issue so I'm not 100% sure of the proper handling. I'm leaning towards the latter.